modules/mauth-authenticator/src/main/java/com/mdsol/mauth/RequestAuthenticator.java

```diff
@@ -37,6 +40,18 @@ public RequestAuthenticator(ClientPublicKeyProvider clientPublicKeyProvider) {
     this(clientPublicKeyProvider, 10L);
   }
 
+  /**
+   * Uses
+   * 10L as default value for request validation timeout
+   * {@link com.mdsol.mauth.utils.ClientPublicKeyProvider} as the EpochTimeProvider
+   *
+   * @param clientPublicKeyProvider PublicKey provider
+   * @param disableV1
```

Comment:
please provide a description for `disableV1` param

Reply:
fixed https://github.com/mdsol/mauth-jvm-clients/pull/89/commits/639fbc9ed0fbf79345c5ddad83d1a8b379b68852

Reply:
Fixed